### PR TITLE
New package: snake

### PR DESCRIPTION
**Pull request by TipzTeam**
I've made a little snake game for Termux.

### CODE REVIEW

**Comment by kcubeterm** [comment]
I think you should use more suitable description .

**Comment by TipzTeam** [comment]
> I think you should use more suitable description .

Let's see what the reviewers say.

**Comment by Grimler91** [comment]
Hi, I like the updated description, very accurate!

Seems the original project is here: https://github.com/gogakoreli/snake/

There are some problems: 
* The original code does not have a license, [basically meaning that it cannot be used for anything](https://opensource.stackexchange.com/questions/1720/what-can-i-assume-if-a-publicly-published-project-has-no-license). I see that you have licensed it as MIT, have you been in contact with @gogakoreli and been told that this is okay?
* The included stdc++.h file is licensed under GPL-3, meaning that the package should be dual licensed, however:
* The included stdc++.h from glibc is not needed. It seems that it is only needed for things like std::cout and std::endl, these are defined in iostream in termux so you can just do `#include <iostream>` to solve it

So, sorry, but unless the original code is licensed correctly we cannot add it to our repositories

**Comment by TipzTeam** [comment]
> Hi, I like the updated description, very accurate!
> 
> Seems the original project is here: https://github.com/gogakoreli/snake/
> 
> There are some problems:
> 
> * The original code does not have a license, [basically meaning that it cannot be used for anything](https://opensource.stackexchange.com/questions/1720/what-can-i-assume-if-a-publicly-published-project-has-no-license). I see that you have licensed it as MIT, have you been in contact with @gogakoreli and been told that this is okay?
> * The included stdc++.h file is licensed under GPL-3, meaning that the package should be dual licensed, however:
> * The included stdc++.h from glibc is not needed. It seems that it is only needed for things like std::cout and std::endl, these are defined in iostream in termux so you can just do `#include <iostream>` to solve it
> 
> So, sorry, but unless the original code is licensed correctly we cannot add it to our repositories

I will work on those issues and I will come back asap

**Comment by TipzTeam** [comment]
- [ ] MIT license issue
- [x] Remove stdc++

Fun fact, this works with clang and g++

**Comment by TipzTeam** [comment]
Also @Grimler91, can I not license this project and be done with it?

**Comment by TipzTeam** [comment]
Actually, I think I can license it in Public domain @Grimler91

**Comment by Grimler91** [comment]
> Actually, I think I can license it in Public domain @Grimler91

It is only public domain if that is stated in the project, or if the original author says so, see https://opensource.stackexchange.com/questions/1720/what-can-i-assume-if-a-publicly-published-project-has-no-license.

File: packages/snake/build.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/TipzTeam/snake-cpp-termux
+TERMUX_PKG_DESCRIPTION="Eat as much as you want while avoiding walls."
+TERMUX_PKG_LICENSE=MIT
+TERMUX_PKG_VERSION=1.0
+TERMUX_PKG_SRCURL=https://github.com/TipzTeam/snake-cpp-termux/archive/${TERMUX_PKG_VERSION}.tar.gz
```
Comment (xeffyr) [inline packages/snake/build.sh:5]: Forks shouldn't be used unless there is a good reason, e.g. original owner transferred project's ownership.

If source requires some changes to get working in Termux, this should be done through patches placed in directory where build.sh is located.

If project doesn't specify a version (https://github.com/gogakoreli/snake/ doesn't specify it), then you should use a date of latest commit in format YYYY.MM.DD and in source URL a git commit hash should be used.

File: packages/snake/build.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+TERMUX_PKG_SHA256=f174e9ef5cd581cbec3748e5d3d0c370dd819858e4126d01cdcb29cff93f6d0b
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_MAINTAINER="Tipz Team @TipzTeam"
+TERMUX_PKG_BUILD_DEPENDS="clang, make"
```
Comment (xeffyr) [inline packages/snake/build.sh:9]: Remove these two from build depends. Packages are cross-compiled by NDK and clang/make are useless here.

**Review by xeffyr — CHANGES_REQUESTED** [review]
Fix build.sh, with resolving license issue pointed by @Grimler91.

**Comment by TipzTeam** [comment]
The original porject is licensed in MIT according to the project owner.

**Comment by TipzTeam** [comment]
It's failing because there's no license file.

**Comment by xeffyr** [comment]
> The original project is licensed in MIT according to the project owner.

Where's that info?

**Comment by TipzTeam** [comment]
> > The original project is licensed in MIT according to the project owner.
> 
> Where's that info?

here: https://twitter.com/team_tipz/status/1302978693862625283

**Comment by TipzTeam** [comment]
Thanks!

**Review by xeffyr — APPROVED** [review]

**Comment by TipzTeam** [comment]
Yay

**Comment by TipzTeam** [comment]
@xeffyr tested in termux, not opening.

**Comment by TipzTeam** [comment]
In build:
"aarch64-linux-android-strip: Unable to recognise the format of the input file `./bin/snake'"

**Comment by TipzTeam** [comment]
it was working with this one: c85ff0e02154c5babb4dbdba0413741a8b420c77

**Pull request merged**